app/assets/stylesheets/geoblacklight/modules/twitter-typeahead.scss

```diff
@@ -0,0 +1,30 @@
+.twitter-typeahead {
+  float: left;
+  width: 100%;
+  z-index: 10000;
+
+  .tt-input.form-control {
```

Comment:
Can we get away with less here?

```scss
.twitter-typeahead {
  input.tt-input.form-control {
    width: 100%;
  }
  input.tt-hint.form-control {
    width: 100%;
  }
}
```

Rather than all of the file?